vcd/datasource_vcd_nsxt_network_imported.go

```diff
@@ -22,10 +22,19 @@ func datasourceVcdNsxtNetworkImported() *schema.Resource {
 					"level. Useful when connected as sysadmin working across different organizations",
 			},
 			"vdc": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				ForceNew:    true,
-				Description: "The name of VDC to use, optional if defined at provider level",
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				Description:   "The name of VDC to use, optional if defined at provider level",
+				Deprecated:    "This field is deprecated in favor of 'owner_id' which supports both - VDC and VDC group IDs",
```

Comment:
```suggestion
				Deprecated:    "This field is deprecated in favor of 'owner_id' which supports both - VDC and VDC Group IDs",
```

Reply:
Changed and I have done a search and replace for "VDC group" -> "VDC Group" which had 30+ occurrences.